### PR TITLE
Update npm packages

### PR DESCRIPTION
**Pull request by thomasguillot**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR updates the different NPM packages to their latest verisons

### How to test the changes in this Pull Request:

1. Does it break anything? 😄

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -132,7 +132,8 @@
     "react-dropzone": "^10.2.1",
     "react-rte": "^0.16.1",
     "recharts": "^1.8.5",
-    "whatwg-fetch": "^3.0.0"
+    "whatwg-fetch": "^3.0.0",
+    "wp-prettier": "git+https://github.com/Automattic/wp-prettier.git#wp-prettier-1.18.2"
```
Comment (adekbadek) [inline package.json:136]: Why is this here? We're [already using `wp-prettier` as `prettier`](https://github.com/Automattic/newspack-plugin/blob/82fc7fddd423021bb22238167d0f6fc54d08813a/package.json#L49)

**Review by adekbadek — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (thomasguillot) [reply package.json:136]: 🤔  no idea, I simply ran `npm update`. Should I just remove the line?

Comment (adekbadek) [reply package.json:136]: Confirmed, that's probably added because of that `prettier` aliasing 🤷‍♀️

**Review by adekbadek — APPROVED** [review]
